CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [Unreleased]
+### Fixed
+- Fixes type annotations for Functions API
```

Comment:
Maybe mention that FunctionHandle is now a new type.

Reply:
Added